app/js/sagas/conceptsSaga.js

```diff
@@ -1,25 +1,47 @@
 import {
   call,
   take,
+  takeEvery,
   fork,
   put,
+  cancel,
 } from 'redux-saga/effects';
 
-import { FETCH_LAB_CONCEPTS } from '../actions/actionTypes';
+import { FETCH_LAB_CONCEPTS, SET_CONCEPT_MEMBER } from '../actions/actionTypes';
 import labConceptsAction from '../actions/labConceptsAction';
 
 import conceptRest from '../rest/concept';
 
-export function* loadSetMemebers(member) {
-  const { uuid } = member;
+export function* loadMemebers(set) {
+  const { uuid } = set.member;
   const response = yield call(conceptRest.getSetMembers, uuid);
```

Comment:
I missed this earlier,  but I think this function should be renamed to getConcept (instead of getSetMembers)... because, unless I'm reading it wrong, this method just fetches a single concept?  In our use case we may call it multiple times to get a bunch of set members, but the function itself could be used any time you want to fetch a concept?

I think the whole concept.js can be moved into react-components into a new file conceptRest.js in the REST directory, and getSetMembers can be renamed getConcept.